### PR TITLE
Adding 8 mentors

### PR DESCRIPTION
**Pull request by niranjannitesh**
Adding 8 mentors:

@harshit98 
@isuruAb
@CosmicCoder96
@royalharsh
@harshkothari410
@jigyasa-grover
@SaptakS
@daminisatya

gh-pages: [http://niteshkumarniranjan.me/gci17.fossasia.org/](http://niteshkumarniranjan.me/gci17.fossasia.org/)

### CODE REVIEW

**Comment by hvardhanx** [comment]
Some of the facebook links redirect to different profiles in your gh-pages link.

**Comment by niranjannitesh** [comment]
I have made changes...

**Comment by niranjannitesh** [comment]
I have written right username but it is showing wrong.

**Comment by abishekvashok** [comment]
@niteshkumarniranjan pls recheck :)

**Comment by niranjannitesh** [comment]
I have figured out the error. the error was in HTML.
In HTML GitHub username was used in facebook username.

**Comment by abishekvashok** [comment]
@niteshkumarniranjan fixes it up in #77

**Comment by niranjannitesh** [comment]
@abishekvashok are you a mentor.

**Comment by abishekvashok** [comment]
@niteshkumarniranjan no i am just a fellow student like you trying to help others :)

**Comment by 0x48piraj** [comment]
@niteshkumarniranjan wait for #77 to be merged :wink: 
Approved your PR @abishekvashok :smile:

**Comment by niranjannitesh** [comment]
@0x48piraj you are a student. Am I right. have you also applied for mentoring.

File: _data/mentors.yml [hunk 1]
```diff
@@ -17,3 +17,55 @@
...
+  facebook: s4pt4ks
+  twitter: Saptak013
+
+
+- name: Damini Satya
+  github: daminisatya
+  image: daminisatya.jpg
+  blog: daminisatya.info
+  facebook: daminisatya
+  twitter: Daminisatya
```
Comment (NaviRocker) [inline _data/mentors.yml]: No new line at the end

**Review by NaviRocker — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -289,17 +289,14 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
     <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
 							<img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
-		                    <img class="col-sm-2" alt="Chahat Bansal" src="img/chahat.png">
+							<img class="col-sm-2" alt="Chahat Bansal" src="img/chahat.png">
```
Comment (abishekvashok) [inline index.html]: Properly intend these lines.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Review by NaviRocker — CHANGES_REQUESTED** [review]
Squash your Commits and keep only the First One

File: index.html (hunk 2 above)
Comment (niranjannitesh) [reply index.html]: I haven't indented this. This is from very earlier.

Comment (abishekvashok) [reply index.html]: Please see the amount of additional spaces added in the giff. If you have done it, then it must be your editor, pls avoid ide's

**Comment by 0x48piraj** [comment]
**Nope. It just depends how deeply are you involved in this community :blush:
I am in love with this community and want to contribute as much I can. :100:** 

> BTW, It's not the place where we should talk personal things. It's not good for the community! 

> Switch to **FOSSASIA's** **[gitter](https://gitter.im/fossasia/fossasia) Channel** :wink:

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Squash the commits! :+1:

**Comment by 0x48piraj** [comment]
@niteshkumarniranjan , Here's a good `gist` on squashing commits : https://gist.github.com/patik/b8a9dc5cd356f9f6f980 :100:

**Comment by NaviRocker** [comment]
Easiest Way to Squash your Commits. Try this,
git reset --soft HEAD~16
git commit -m "<Your commit message>"
git push -f origin gh-pages

**Comment by niranjannitesh** [comment]
@naveentherocker @0x48piraj I had already Squashed my Commits.

**Comment by 0x48piraj** [comment]
Good Job fellas! :1st_place_medal:

**Comment by NaviRocker** [comment]
Now Looking Fine. If you can update the Preview Website, it's better.
image

**Comment by niranjannitesh** [comment]
This has been done mistakenly I have used beautify in index.html that's why it happened. @naveentherocker

**Comment by niranjannitesh** [comment]
@naveentherocker take a look

**Review by ParthS007 — APPROVED** [review]
LGTM ! Good Work :100:

**Pull request merged**